packages/styling/src/styles/theme.ts

```diff
@@ -101,8 +101,8 @@ function _loadFonts(theme: ITheme): { [name: string]: string } {
     for (const propName of Object.keys(font)) {
       const name = fontName + propName.charAt(0).toUpperCase() + propName.slice(1);
       let value = font[propName];
-      if (propName === 'fontSize' && value && value.indexOf && value.indexOf('px') < 0) {
-        value += 'px';
+      if (propName === 'fontSize' && typeof value === 'number') {
```

Comment:
Why don't we need to make this change in Fabric 7 too?

Reply:
i did

Reply:
I don't want to be a nuisance, but does this handle all edge cases?
`number` 16 => '16px' ✅
`string` '16px' => '16px' ✅
`string` '16' => '16' ❓
Is this last case possible and handled somewhere?

Reply:
Or would that not be valid according the published API? 
I.e. are users supposed to enter either:
`number` 16
or
`string` '16px'
(or maybe even `string` '16pt' ?)

Reply:
Yeah I would argue string 16 is not valid. If you pass in a number, we will automatically treat that as pixels. If it's a string, then we won't mess with your value and we assume you know what you're doing. And yes, it's correct that you would use a string for non-px units.
Being nitpicky is good, thanks for reviewing =)